state/protocol/seed/seed.go

```diff
@@ -11,9 +11,11 @@ import (
 // FromParentSignature reads the raw random seed from a QC sigData.
 // The sigData is an RLP encoded structure that is part of QuorumCertificate.
 // The indices can be used to  generate task-specific seeds from the same signature.
+// TODO: this only works for consensus V2 where a V2 packer is used to unpack the sig data
```

Comment:
same thing, it will work for V3 exactly the same, unless I am missing something very obvious.